### PR TITLE
Two new pages added with images and code blocks

### PR DESCRIPTION
**Pull request by Hikkigaya**
### Summary

Two new pages added with images and code blocks

### Related PRs, issues, or features (optional)

- 244

### Metadata

- [x] Adds new file(s)
- [x] Edits existing file(s)
- [ ] Removes file(s)

### PR contributor checklist

- [x] My PR follows the [Kobiton Docs contributor guidelines](https://github.com/kobiton/docs/blob/main/CONTRIBUTE.adoc), meaning:

    - My content contains correct spelling and grammar.
    - My directories and files are [named](https://github.com/kobiton/docs/blob/main/CONTRIBUTING.md#directory-and-file-names) and [structured](https://github.com/kobiton/docs/blob/main/CONTRIBUTING.md#directory-structure) correctly.
    - My style and voice follows the [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/brand-voice-above-all-simple-human).
    - I added all new pages to their section's [`nav.adoc` file](https://github.com/kobiton/docs/blob/main/CONTRIBUTING.md#configure-navigation-in-navadoc).
    - I removed all localizations (like `en-us`) from my URLs.

### CODE REVIEW

**Comment by tungmhoang** [comment]
@erincbailey you can review this PR at:
- http://10.2.7.208/integrations/applitools-eyes/applitools-eyes-integration
- http://10.2.7.208/automation-testing/scripting/run-applitools-eyes-script
- http://10.2.7.208/session-explorer/manage-sessions

**Review by tungmhoang — APPROVED** [review]

File: docs/modules/integrations/pages/applitools-eyes/applitools-eyes-integration.adoc [hunk 1]
```diff
@@ -0,0 +1,58 @@
+= Enable Applitools Eyes integration
```
Comment (erincbailey) [inline docs/modules/integrations/pages/applitools-eyes/applitools-eyes-integration.adoc:1]: Remove "Enable" from title. Looking at all the other integration pages, this isn't part of the title

File: docs/modules/automation-testing/pages/scripting/run-applitools-eyes-script.adoc [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+* Additionally, replace `YOUR_API_KEY` below with the runner API key from Applitools.
++
+[source,javascript]
+----
+// Initialize the eyes SDK and set your private API key.
+Eyes eyes = new Eyes();
+eyes.setApiKey("YOUR_API_KEY");
+----
+
+At the time of writing, the final test script would look like the below:
```
Comment (erincbailey) [inline docs/modules/automation-testing/pages/scripting/run-applitools-eyes-script.adoc:81]: Remove "At the time of writing," and make the capital "The final..."
(The phrase is not needed and instills doubt. All tech docs are valid as of the time of writing.)

**Review by erincbailey — CHANGES_REQUESTED** [review]
Just two minor comments in the pages.
Plus, should the images be left justified? Sampling other pages - they are left justified. Please confirm.

**Comment by tungmhoang** [comment]
@erincbailey feedback addressed. Updated images to be left justified as well. @Hikkigaya please note that images should be left-justified, unless it is a mobile screenshot. Mobile screenshot just looks weird if left-justified.

**Comment by Hikkigaya** [comment]
> @erincbailey feedback addressed. Updated images to be left justified as well. @Hikkigaya please note that images should be left-justified, unless it is a mobile screenshot. Mobile screenshot just looks weird if left-justified.

Noted

**Review by erincbailey — APPROVED** [review]
approved

**Pull request merged**